### ISSUE TITLE
PReLU?

### ISSUE THREAD

**Issue by Avmb** [marian-nmt/marian-dev#232]
Shouldn't alpha be an expression with the same dimension as the input, rather than a float? https://github.com/marian-nmt/marian-dev/blob/2e77d5ecebcf48dd4114878cc9c136a7d09c9823/src/graph/expression_operators.cpp#L31

**Comment by emjotde** [comment]
@snukky ?

**Comment by snukky** [comment]
Yes, I think alpha should be learnable parameters to get the actual PReLU. What we have is basically a parametrizable leaky ReLU.

**Comment by emjotde** [comment]
I think you would need to model this after the way layer_norm is done.

**Comment by emjotde** [comment]
@snukky We should either fix that or remove the node. Is that still a problem?
